inc/commondbtm.class.php

```diff
@@ -5143,4 +5143,55 @@ private function assetBusinessRules($condition) {
          }
       }
    }
+
+   /**
+    * Get incidents, request, changes and problem linked to this object
+    *
+    * @return array
+    */
+   public function getITILTickets(bool $count = false) {
```

Comment:
Not sure this is the right place. Seems too large for an ITIL specific method

Reply:
Where should we define this ? @orthagh @cedric-anne

Reply:
It could be a trait but it would require to add this trait to lots of elements. So maybe in CommonDBTM it is a good place.